File: Mahesh_Babar/running_sum_of_1d_array.java
// https://leetcode.com/problems/running-sum-of-1d-array/

import java.util.Arrays;

public class running_sum_of_1d_array {
    public static void main(String[] args) {

        System.out.println(Arrays.toString(runningSum(new int[] { 1, 2, 3, 4, 5 })));
    }

    private static int[] runningSum(int[] nums) {
        int res[] = new int[nums.length];

        int runningSum = 0;
        for (int i = 0; i < nums.length; i++) {
            runningSum += nums[i];
            res[i] = runningSum;
        }
        return res;
    }
}

File: Simran_Nagdeo/Running Sum of 1d Array.js
var runningSum = function(nums) {
   
    let arr1=[];
   for(let i=0;i<nums.length;i++){
       let sum=0;
       for(let j=0;j<=i;j++){
           sum=sum+nums[j];
       }
       arr1.push(sum);
   }
   return arr1;

};
